StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/StripePaymentSheet+Imports.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  StripePaymentSheet+Imports.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 5/15/23.
+//
+
+// Used to organize exported modules from StripePaymentSheet
+@_exported import StripePayments
```

Comment:
I looked and found this file already exists as `StripePaymentSheet+Exports.swift`

Reply:
TIL